web/src/stream_popover.js

```diff
@@ -566,9 +570,53 @@ export async function build_move_topic_to_stream_popover(
         event.stopPropagation();
     }
 
+    function update_warning_banner() {
+        // The warning banner only shows when
+        // 1) submit button is NOT disabled, and
+        // 2) target topic is an existing topic, and
+        // 3) propagate_mode is NOT change_one
+
+        // submit button
+        if ($("#move_topic_modal .dialog_submit_button")[0].disabled) {
+            $("#move_topic_modal .move_topic_form_warning_container").html("");
+            return;
+        }
+
+        // target topic
+        if (stream_widget_value !== undefined) {
+            const stream_topics = stream_topic_history.get_recent_topic_names(
+                Number.parseInt(stream_widget_value, 10),
+            );
+            const {new_topic_name} = get_params_from_form();
+            if (!stream_topics.includes(new_topic_name.trim())) {
```

Comment:
Is this comparison case-insensitive? Zulip topics are.

Reply:
So I use localeCompare with array.some but it may be overkill.